src/pages/dashboard.tsx

```diff
@@ -1318,35 +1990,177 @@ const Dashboard = () => {
+                                            " font-size-18"
+                                          }
+                                        >
+                                          <i
+                                            className={
+                                              CoinClassNames[
+                                                EventMap[
+                                                  asset.collateralMarket.toUpperCase()
+                                                ]
+                                              ] ||
+                                              asset.collateralMarket.toUpperCase()
+                                            }
+                                          />
+                                        </span>
+                                      </div>
+                                      <span>
+                                        {
+                                          EventMap[
+                                            asset.collateralMarket.toUpperCase()
+                                          ]
+                                        }
+                                      </span>
+                                    </div>
+                                  </th>
+                                  <td>
+                                    <div className="text-muted">
+                                      {BNtoNum(Number(asset.collateralAmount))}
+                                    </div>
+                                  </td>
+                                  <td>
+                                    <Button
+                                      className="text-muted"
+                                      color="light"
+                                      outline
+                                      onClick={() => {
+                                        handleLiquidation(asset)
+                                      }}
+                                    >
+                                      {(isTransactionDone && asset.isLiquidationDone) ? (
+                                        <Spinner>Loading...</Spinner>
+                                      ) : (
+                                        "Liquidate"
+                                      )}
+                                    </Button>
+                                  </td>
+                                  {/* <td>
+                    <div className="text-muted">{Number(asset.acquiredYield).toFixed(3)}</div>
+                  </td>  */}
+                                </tr>
+                              ))
+                            ) : (
+                              <tr align="center">
+                                <td colSpan={5}>No Records Found.</td>
+                              </tr>
+                            )}
+                          </tbody>
+                        </Table>
+                        <Button
```

Comment:
Positioning of this button needs improvement.

Reply:
Ok